docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,552 @@
-    The following example illustrats part of the [Tizen.NUI.ColorVisual](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.ColorVisual.html) class, which defines a color visual property map:
 
-    ```
-    public class ColorVisual : VisualMap
-
-    private Color _mixColorForColorVisual = null;
-
-    public Color Color
-    {
-        get
-        {
-            return _mixColorForColorVisual;
-        }
-        set
-        {
-            _mixColorForColorVisual = value;
-            UpdateVisual();
-        }
-    }
-    ```
 
--   You can create a visual from an output visual map:
+<!---------------------------------------------------------------------------------------------------------------------------->
+<a name="visualMap"></a>
+## VisualMap encapsulates common properties of Visuals 
 
-    ```
-    var colorMap = new ColorVisual{Color=Color.White;};
-    var _colorVisual = VisualFactory.Instance.CreateVisual(colorMap.OutputVisualMap);
-    RegisterVisual(ColorVisualPropertyIndex, _colorVisual);
-    ```
+The VisualMap is a base class for all Visuals.
 
-    You can also create a visual using an output map in a control property. The following example uses the `Background` property:
+**Table: The VisualMap properties - all are optional.**
 
-    ```
-    ColorVisual colorVisualMap1 = new ColorVisual();
-    colorVisualMap1.Color = Color.Green;
-    _visualView.Background = colorVisualMap1.OutputVisualMap;
+| Property            | Type                        | Default Value                    | Description                                                                                                       |
+|---------------------|-----------------------------|----------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| `Size`              | `Size2D`                    | `(1, 1)`                         | The Visual size - depending on the `SizePolicy` value is interpreted as the relative or the absolute size.        |
+| `RelativeSize`      | `RelativeVector2`           | `(1.0f, 1.0f)`                   | The Visual size - use to specify the relative size as a percentage of the parent.                                 |
+| `SizePolicy`<br>`SizePolicyWidth`<br>`SizePolicyHeight` | `VisualTransformPolicyType` | `Relative`   | Specifies whether the Visual size/width/height is relative (`Relative`) or absolute (`Absolute`). |
+| `Position`          | `Vector2`                   | `(0.0f, 0.0f)`                   | The Visual's offset (absolute or relative - percentage) specified as the shift of the Visual reference point with respect to the parent's reference point (values on x and y axes increase towards right and bottom, respectively). |
+| `RelativePosition`  | `RelativeVector2`           | `(0.0f, 0.0f)`                   | The Visual's offset specified as a percentage of the parent.                                                      |
```

Comment:
and now? :
The visual offset specified as a percentage of the size of the parent. 

it's like Position, but it is given in relative units instead of in absolute ones